tools/rime_api_console.cc

```diff
@@ -205,9 +214,8 @@ int main(int argc, char* argv[]) {
     rime->join_maintenance_thread();
   fprintf(stderr, "ready.\n");
 
-  RimeSessionId session_id = rime->create_session();
-  if (!session_id) {
-    fprintf(stderr, "Error creating rime session.\n");
+  RimeSessionId session_id;
+  if (!ensure_session(rime, session_id)) {
```

Comment:
Just write here
``` c++
RimeSessionId session_id = 0;
```
The initial session can be created by the call in the while loop below, because `find_session(0)` should return false.